contents/blog/stop-treating-engineers-differently.md

```diff
@@ -0,0 +1,68 @@
+---
+date: 2022-09-07
+title: "Startups, please stop treating engineers like a different kind of person"
```

Comment:
```suggestion
title: "Hey startups, stop treating engineers like a different kind of person!"
```

Reply:
I agree the title could be better, but I worry your suggestion reads slightly _too_ clickbaity/LinkedIn influencery. What about: 'Startups, can we please stop treating engineers like a different kind of person?'

Reply:
Maybe we just go with 'Stop treating engineers like a different kind of person' then? It's tight and to the point.

Reply:
I really want 'startups' in there though if poss - thinking of the HN premortem where people complain this is only relevant to startups, and also I think it'll attract more clicks from the similarly enraged startupy people that this is written for. 

How about 'Startups, stop treating engineers like a different kind of person.'?

Reply:
works for me

Reply:
Random last minute suggestion as I want to merge this today...

"Startups, stop treating engineers like a different species"

or

"Startups, stop treating engineers like a different species of human"

Reply:
Let's go with your first one! (It's snappier)